### PR TITLE
Check if store file exists when registering primary

### PR DESCRIPTION
**Pull request by aolfat**
# Description

### Select type(s) of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

# Checklist:

- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have fixed any merge conflicts

### CODE REVIEW

File: provider/k8s.go [hunk 1]
```diff
@@ -676,6 +676,24 @@ func (k8s *K8sOfflineStore) RegisterPrimaryFromSourceTable(id ResourceID, source
 }
 
 func blobRegisterPrimary(id ResourceID, sourcePath string, logger *zap.SugaredLogger, store FileStore) (PrimaryTable, error) {
+	sourceFilePath, err := filestore.NewEmptyFilepath(store.FilestoreType())
```
Comment (aolfat) [inline provider/k8s.go:679]: this is basically what you had erik i just added the exists at the end

File: provider/store_utils_test.go [hunk 2]
```diff
@@ -0,0 +1,139 @@
+package provider
+
+import (
```
Comment (aolfat) [inline provider/store_utils_test.go:3]: this was just a nice refactor from the other pr

File: provider/k8s_test.go [hunk 3]
```diff
@@ -124,58 +124,11 @@ func TestBlobInterfaces(t *testing.T) {
 		"Test File Upload and Download": testFileUploadAndDownload,
 	}
 
-	err := godotenv.Load("../.env")
```
Comment (aolfat) [inline provider/k8s_test.go:127]: i'll even remove these changes from the pr

File: provider/filestore_test.go [hunk 4]
```diff
@@ -1,314 +0,0 @@
...
-	"io/ioutil"
-	"os"
-	"testing"
-
-	"github.com/joho/godotenv"
-
-	pc "github.com/featureform/provider/provider_config"
-)
-
-func TestFileStore(t *testing.T) {
```
Comment (aolfat) [inline provider/filestore_test.go:22]: this file doesn't even make sense for your stuff anymore. We probably will eventually need filestore tests but for now

**Review by epps — APPROVED** [review]
🙌

**Pull request merged**